stable/app/Chart.yaml

```diff
@@ -15,4 +15,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.5.4
+version: 0.5.5
```

Comment:
Since it's a new feature, shouldn't we bump "minor"?

Reply:
Previous additions also bumped the patch version. 
I am okay with either.
Will bump to 0.6.0

Reply:
yeah !